armi/settings/caseSettings.py

```diff
@@ -14,15 +14,12 @@
 
 """
 This defines a Settings object that acts mostly like a dictionary. It
-is meant to be treated mostly like a singleton, where each custom ARMI
-object has access to it. It contains global user settings like the core
-power level, the input file names, the number of cycles to run, the run type,
-the environment setup, and hundreds of other things.
+is meant so that each ARMI run has one-and-only-one Settings object. It global
```

Comment:
Missing something in the second sentence on this line

Reply:
Fixed